### PR TITLE
Create extra Travis job to clone and test the starter pack

### PR DESCRIPTION
**Pull request by MetcalfeTom**
**Proposed changes**:
- related to https://github.com/RasaHQ/rasa_core/issues/1427
- Add an extra stage to the travis script which clones the [NLU starter pack](https://github.com/RasaHQ/starter-pack-rasa-nlu) and tests it upon successful NLU build

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [x] added some tests for the functionality
- [ ] updated the documentation
- [x] updated the changelog

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -80,6 +80,18 @@ jobs:
     - cd rasa-website
     - git commit --allow-empty -m "trigger nlu docs update"
     - git push origin master
+  - stage: Test NLU starter pack
+    python: 3.6
+    script:
+    - git clone https://github.com/RasaHQ/starter-pack-rasa-nlu.git
+    - cd starter-pack-rasa-nlu
+    - py.test test_nlu.py
```
Comment (tmbo) [inline .travis.yml]: can we put the `test_nlu.py` into a folder, e.g. `tests` and run `py.test tests`?

File: .travis.yml [hunk 2]
```diff
@@ -80,6 +80,18 @@ jobs:
...
+  - stage: Test stack starter pack (NLU only)
+    python: 3.6
+    script:
+    - git clone https://github.com/RasaHQ/starter-pack-rasa-stack.git
+    - cd starter-pack-rasa-stack
+    - py.test test_stack.py -k test_nlu_interpreter
```
Comment (tmbo) [inline .travis.yml]: same here, how about putting the tests into `tests/nlu` and using that dir here?

**Comment by tmbo** [comment]
Can you please change this to run the stages in parallel (the same way I did in core)

**Comment by MetcalfeTom** [comment]
No, because we don't do the same integration tests in `rasa_nlu` as in `rasa_core`, [we only test the repo and deploy](https://travis-ci.com/RasaHQ/rasa_nlu/builds/96274557)

**Comment by MetcalfeTom** [comment]
@tmbo we can test the nlu components of both packs in parallel, with an extra cost of 90 seconds build time

**Pull request merged**